distribution/docs/src/main/resources/content/_quickstart/quickstart-installing.adoc

```diff
@@ -186,10 +186,13 @@ ${branding-lowercase}${at-symbol}local>
 
 If ${branding} is being installed on a remote server that has no user interface some additional steps must be taken prior to starting the system.
 
-. Update any references to localhost in the following files. These references to localhost should be updated to match either the hostname or IP of the system.
-** `${home_directory}/etc/custom.system.properties`
-** `${home_directory}/etc/users.properties`
-** `${home_directory}/etc/users.attributes`
+. Update the `${home_directory}/etc/custom.system.properties` file.
+.. The entry `org.codice.ddf.system.hostname=localhost` should be updated to match either the hostname or IP of the system.
```

Comment:
✏️ Clarify how it should be updated like you have for the other 2 steps below, because the `org.codice.ddf.system.hostname=` part of that line should not change. E.g. 

>`The value for org.codice.ddf.system.hostname should be the hostname or IP of the system.`

or

>Change `org.codice.ddf.system.hostname=localhost` to `org.codice.ddf.system.hostname=<HOSTNAME>` or `org.codice.ddf.system.hostname=<IP>`.